config/gen2convert.yaml

```diff
@@ -37,18 +37,21 @@ collections:
     -
       pattern: "^(.+)/rerun/"
       repl: "shared/"
+    -
+      pattern: "^(.+)/CALIB"
+      repl : "calib"
   overrides:
     # A dictionary of DatasetType -> Collection mappings that force all
     # Datasets with that DatasetType into a particular Collection.
     # Collection names can be str.format patterns that utilize any of
     # the Gen3 DataUnits or Gen2 DataIds associated with the Dataset.
-    raw: raw/{instrument}
+    raw: raw
     ref_cat: ref/{name}
     ref_cat_config: ref/{name}
-    transmission_filter: transmission/{instrument}
-    transmission_sensor: transmission/{instrument}
-    transmission_optics: transmission/{instrument}
-    transmission_atmosphere: transmission/{instrument}
+    transmission_filter: calib
+    transmission_sensor: calib
+    transmission_optics: calib
+    transmission_atmosphere: calib
```

Comment:
This is only needed for the gen2 conversion, correct?  Specifically, once fully switched to gen3, all of this information will be in the butler repository, and will not need yaml files to special case things?

Reply:
Correct; the idea is that eventually obs_ packages will provide some kind of script that dumps these datasets into the right collection in a butler repository, including validity ranges for all of them.  I don't know exactly what those scripts (or their source data) will look like yet.